src/components/project/Releases.vue

```diff
@@ -0,0 +1,380 @@
+                >
+                  <span class="project-releases__file-name">{{ file.originFileName }}</span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        v-else
+        class="project-releases__empty"
+      >
+        <div class="project-releases__empty-title">{{ $t('components.releases.empty.title') }}</div>
+        <div class="project-releases__empty-description">
+          {{ $t('components.releases.empty.description') }}
+          <ul class="project-releases__empty-list">
+            <li>{{ $t('components.releases.empty.benefits.sourceMaps') }}</li>
+            <li>{{ $t('components.releases.empty.benefits.identify') }}</li>
+          </ul>
+          <a href="https://docs.hawk.so/releases" target="_blank" class="project-releases__empty-link">
+            {{ $t('components.releases.empty.learnMore') }}
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { FETCH_PROJECT_RELEASES } from '@/store/modules/projects/actionTypes';
+import DetailsSuspectedCommits from '../event/details/DetailsSuspectedCommits.vue';
+
+export default {
+  name: 'ProjectReleases',
+  components: {
+    DetailsSuspectedCommits
+  },
+  props: {
+    projectId: {
+      type: String,
+      required: true
+    }
+  },
+  data() {
+    return {
+      /**
+       * Tracks which releases are expanded by index
+       */
+      expandedReleases: {},
+    };
+  },
+  computed: {
+    /**
+     * Current viewed project
+     */
+    project() {
+      return this.$store.getters.getProjectById(this.projectId);
+    },
+
+    /**
+     * Project releases from Vuex store
+     */
+    releases() {
+      return this.project?.releases || [];
+    },
+
+    /**
+     * Groups releases by day and sorts them by timestamp
+     * - Groups releases into 'today', 'yesterday', and date-based categories
+     * - Sorts groups with 'today' first, then 'yesterday', then by date
+     * - Sorts releases within each group by timestamp (newest first)
+     * @returns {Object} Object with days as keys and sorted releases as values
+     */
+    groupedReleases() {
+      const groups = {};
+      
+      if (!this.releases) return groups;
+
+      const now = new Date();
+      const today = new Date(now.getFullYear(), now.getMonth(), now.getDate());
+      const yesterday = new Date(today);
+      yesterday.setDate(yesterday.getDate() - 1);
+
+      this.releases.forEach(release => {
+        const timestamp = this.getTimestampFromReleaseId(release.id) * 1000;
+        const date = new Date(timestamp);
+        const releaseDay = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+        let day;
+
+        if (releaseDay.getTime() === today.getTime()) {
+          day = 'today';
+        } else if (releaseDay.getTime() === yesterday.getTime()) {
+          day = 'yesterday';
+        } else {
```

Comment:
seems redundant. `prettyDate`filter works with dates and formants 'Today' and 'Yesterday' internally